cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -99,24 +118,24 @@ func (g *GetEventsRequest) Valid(maxLimit uint) error {
 
 	// Validate the paging limit (if it exists)
 	if g.Pagination != nil && g.Pagination.Cursor != nil {
-		if g.StartLedger != 0 {
-			return errors.New("startLedger and cursor cannot both be set")
+		if g.StartLedger != 0 || g.EndLedger != 0 {
+			return errors.New("ledger ranges and cursor cannot both be set") //nolint:forbidigo
```

Comment:
is the `nolint` still needed?

Reply:
Yep still needed for now as` errors.New` is coming from our internal /error package.